eng/common/pipelines/templates/steps/docker-pull-image.yml

```diff
@@ -10,15 +10,11 @@ parameters:
   - name: Tag
     type: string
     default: 'latest'
-  - name: ImageIdVar
-    type: string
-
 steps:
 - pwsh: |
     docker login $(containerRegistry) -u "${{ parameters.ContainerRegistryClientId }}" -p "${{ parameters.ContainerRegistryClientSecret }}"
   displayName: Login container registry
 - pwsh: |
     $imageId = '${{ parameters.ContainerRegistry }}/${{ parameters.ImageName }}:${{ parameters.Tag }}'
```

Comment:
If we are going to duplicate this id combining in other places we should consider taking in only one parameter for the full id that way we can consistently control the full string in one variable. You might need to parse the container registry name out for the login step but that is probably better than duplicating the image name parts.